tools/sklearn/ml_visualizations.xml

```diff
@@ -0,0 +1,121 @@
+<tool id="ml_visualizations" name="Machine Learning Visualizations" version="@VERSION@">
+    <description>include various types of plotting for machine learning</description>
```

Comment:
This tool name may be too ambitious. A more proper name is appreciated. @anuprulez @bgruening